src/IotHub/IotHub/help/Invoke-AzIotHubManualFailover.md

```diff
@@ -0,0 +1,183 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.IotHub.dll-Help.xml
+Module Name: Az.IotHub
+online version: https://docs.microsoft.com/en-us/powershell/module/az.iothub/invoke-aziothubmanualfailover
+schema: 2.0.0
+---
+
+# Invoke-AzIotHubManualFailover
+
+## SYNOPSIS
+Initiate a manual failover for the IoT Hub to the geo-paired disaster recovery region.
+
+## SYNTAX
+
+### ResourceSet (Default)
```

Comment:
`-WhatIf` and `-Confirm` should be added to *syntax* and *parameters*

Reply:
Updated help file